source/_components/camera.generic.markdown

```diff
@@ -65,7 +65,7 @@ framerate:
   required: false
   type: integer
 verify_ssl:
-  description: Enable or disable SSL certificate verification.
+  description: Enable or disable SSL certificate verification.  Disable this to use a http-only camera or if you have a self-signed or invalid SSL certificate.
```

Comment:
_Set to `false` to use an http-only camera, or you have a self-signed SSL certificate and haven't installed the CA certificate to enable verification._

No need to mention invalid SSL certificates IMO

Reply:
Why not?  If people are using https to connect to 127.0.0.1:8123 they'll have one.

Reply:
Oh, gotcha.  That works :)